<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultWindow</class>
 <widget class="QDialog" name="resultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="imageBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="resultNowbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
